README.md

```diff
@@ -56,10 +64,10 @@ Please also check [Vue Storefront requirements](https://docs.vuestorefront.io/gu
 
 | Package       | Version        |
 |:-------------:|:--------------:|
-| PHP           |  7.3 (>= 7.2)  |
+| PHP           |  7.4 (>= 7.3)  |
```

Comment:
Are you sure the 7.3 >= is not valid?

Reply:
I don't understand. For now we recommend 7.4, but it has to be greater than 7.3.

Reply:
Ahhh ok. My bad, it's fine.